assets/css/style.css

```diff
@@ -196,6 +334,7 @@ html[dir='rtl'] .apertiumLogo {
     }
 }
 
+
```

Comment:
I'm not sure that adding all this extra space adds much. Could you explain your reasoning?